src/core/shared/response/share.dto.ts

```diff
@@ -6,4 +6,5 @@ export class ShareResDto {
   public exception?: string;
   public page?: number;
   public pageSize?: number;
+  public totalPages?: number;
```

Comment:
Missing those types: 
totalDocs {Number} - Total number of documents in collection that match a query.
prevPage {Number} - Previous page number if available or NULL
nextPage {Number} - Next page number if available or NULL